xcode/Subconscious/Shared/Components/Common/EntryRow.swift

```diff
@@ -11,42 +11,29 @@ import SwiftUI
 /// Provides a preview/excerpt of the entry.
 struct EntryRow: View, Equatable {
```

Comment:
What changed: removed title, refactored list style to work best with

- Excerpt
- Slug
- Date